hd/etc/welcome.txt

```diff
@@ -286,11 +289,11 @@ fr: Les <strong>droits des magiciens</strong> sont actuellement <strong>suspendu
         </div>
       </div>
     %end;
-    %if;(plugins!="" and b.plugins!="")
+    %if;(plugins!="" and b.plugin!="")
```

Comment:
why to change b.plugins by b.plugin ?
Similar question few lines below, why to change b.plugins by plugins ?

Reply:
I believe it should be b.plugins everywhere.
The .gwf parameter gives the list of comma separated active plugins (base specific)
%plugins; gives the list of plugins loaded at gwd start (server specific)